pages/api/user/[id].js

```diff
@@ -7,7 +7,7 @@ export default async function handler(request, response) {
   const { id } = request.query;
 
   const user = await User.findById(id).populate({
-    path: "favoritePlaces",
+    path: "favoritePlaces createdPlaces",
```

Comment:
the path shouldn't contain white spaces. 
Do you mean:
```suggestion
    path: "createdPlaces",
```

Reply:
Okay, but i need this to populate favoritePlaces and createdPlaces.

Reply:
I changed the code to this, i don't have a better solution at the moment...

`const user = await User.findById(id)
    .populate({
      path: "favoritePlaces",
      populate: { path: "reviews activities" },
    })
    .populate({
      path: "createdPlaces",
      populate: { path: "reviews activities" },
    });`